distributed/taxbrain_server/_version.py

```diff
@@ -39,19 +39,19 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False):
             if e.errno == errno.ENOENT:
                 continue
             if verbose:
-                print("unable to run %s" % args[0])
+                print(("unable to run %s" % args[0]))
```

Comment:
As we're moving to Python 3, maybe it would be better to use the new `format(…)` syntax which is now preferred over `%`?

Reply:
Right, I wound up deleting this file. I don't think there are any other places where strings are formatted.